src/index.js

```diff
@@ -0,0 +1,123 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+
+import '@ovh-ux/ng-translate-async-loader';
+
+import ChatbotCtrl from './controller';
+import ChatbotServiceProvider from './service';
+import directive from './directive';
+
+import iconsTemplate from './templates/icons.html';
+import embeddedTemplate from './templates/embedded.html';
+
+import './index.less';
+
+const moduleName = 'ngOvhChatbot';
+
+angular
+  .module(moduleName, [
+    'ngAnimate',
+    'ngEmbed',
+    'ngTranslateAsyncLoader',
+    'ovh-angular-user-pref',
+    translate,
+  ])
+  .controller('ChatbotController', ChatbotCtrl)
+  .provider('ChatbotService', ChatbotServiceProvider)
+  .directive('ovhChatbot', directive)
+  .run(($templateCache) => {
+    $templateCache.put('ng-ovh-chatbot-icons-template.html', iconsTemplate);
+    $templateCache.put('ng-ovh-chatbot-embedded-template.html', embeddedTemplate);
+  })
+  .run(/* @ngTranslationsInject ./translations */)
+  .constant('CHATBOT_MESSAGE_TYPES', {
```

Comment:
`constants` should be move in `contants.js`

Reply:
Agree.